src/flag/model.rs

```diff
@@ -124,3 +204,45 @@ fn convert_to_static<'func>(flag: &'func Flag) -> Result<Flag<'static>, ()> {
         &_ => Err(()),
     }
 }
+
+#[cfg(test)]
+mod tests {
+
```

Comment:
Why adding a new line here? It does not really fit the global style of the code base.